### PR TITLE
Inspect Object

### PR DESCRIPTION
**Pull request by adfost**
Adds a button to inspect an object directly
Screen Shot 2022-03-03 at 5 52 00 PM
.

### CODE REVIEW

File: portal-ui/src/screens/Console/Tools/KeyRevealer.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  distributedSetup: state.system.distributedSetup,
+});
+
+const mapDispatchToProps = {
+  setErrorSnackMessage,
+};
+
+const connector = connect(mapState, mapDispatchToProps);
+
+export default connector(KeyRevealer);
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Tools/KeyRevealer.tsx:60]: Please remove the `setErrorSnackMessage` , `mapState`, `connect` as it is not required in this component.
```suggestion
export default KeyRevealer;
```

File: portal-ui/src/screens/Console/Tools/KeyRevealer.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { Box, Button } from "@mui/material";
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Tools/KeyRevealer.tsx:1]: Please add License Header

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/InspectObject.tsx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+                    description=""
+                  />
+              <Grid item xs={12} className={classes.modalButtonBar}>
+                <Button
+                    type="submit"
+                    variant="contained"
+                    color="primary"
+                    onClick={performInspect}
+                >
+                  Save
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/InspectObject.tsx:147]: The button text could be changed to `Inspect`
```suggestion
                  Inspect
```

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/InspectObject.tsx [hunk 4]
```diff
@@ -0,0 +1,181 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2021 MinIO, Inc.
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/InspectObject.tsx:2]:
```suggestion
// Copyright (c) 2022 MinIO, Inc.
```

**Review by prakashsvmx — CHANGES_REQUESTED** [review]
:+1:  
Tested. Minor comments.

**Review by cniackz — CHANGES_REQUESTED** [review]
@adfost there are console red messages (warning) at `inspect` feature:

https://user-images.githubusercontent.com/6667358/156887189-3ee73fbe-8410-4081-83c4-27a5aa0942a2.mov
[console-log.log](https://github.com/minio/console/files/8190827/console-log.log)

# Main message
```js
react_devtools_backend.js:4061 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
```

# Steps or flow of the failure:
1. Inspect with no encryption
2. Then inspect with encryption <--- It fails

Encrypting alone is working
Please fix it!

**Review by cniackz — CHANGES_REQUESTED** [review]
Hello @adfost 

There is a new `warning` message:

Screen Shot 2022-03-07 at 8 35 56 AM

# Steps to reproduce:
1. Get the PR changes locally
2. Then create a folder
3. Upload an image
4. Inspect the image
5. Answer `No` to modal's question: `Would you like to encrypt Screen Shot 2022-03-07 at 8.35.35 AM.png?`
6. Click `Inspect`

# Warning message:
```js
react_devtools_backend.js:4061 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    at InspectObject (http://localhost:5005/static/js/src_screens_Console_Buckets_BucketDetails_BrowserHandler_tsx.chunk.js:1115:5)
    at ConnectFunction (http://localhost:5005/static/js/bundle.js:121985:68)
    at WithStyles (http://localhost:5005/static/js/bundle.js:88998:23)
overrideMethod @ react_devtools_backend.js:4061
printWarning @ react-dom.development.js:68
error @ react-dom.development.js:44
warnAboutUpdateOnUnmountedFiberInDEV @ react-dom.development.js:23915
scheduleUpdateOnFiber @ react-dom.development.js:21841
dispatchAction @ react-dom.development.js:16140
(anonymous) @ InspectObject.tsx:97
Promise.then (async)
performInspect @ InspectObject.tsx:76
callCallback @ react-dom.development.js:3946
invokeGuardedCallbackDev @ react-dom.development.js:3995
invokeGuardedCallback @ react-dom.development.js:4057
invokeGuardedCallbackAndCatchFirstError @ react-dom.development.js:4071
executeDispatch @ react-dom.development.js:8244
processDispatchQueueItemsInOrder @ react-dom.development.js:8276
processDispatchQueue @ react-dom.development.js:8289
dispatchEventsForPlugins @ react-dom.development.js:8300
(anonymous) @ react-dom.development.js:8509
batchedEventUpdates$1 @ react-dom.development.js:22397
batchedEventUpdates @ react-dom.development.js:3746
dispatchEventForPluginEventSystem @ react-dom.development.js:8508
attemptToDispatchEvent @ react-dom.development.js:6006
dispatchEvent @ react-dom.development.js:5925
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11277
discreteUpdates$1 @ react-dom.development.js:22414
discreteUpdates @ react-dom.development.js:3757
dispatchDiscreteEvent @ react-dom.development.js:5890
```

Please fix this new `warning` message.
And thanks! 👍 

And in the case this is not related to this change but an existing issue from other component; then I think we can open new issue to handle this fix there and then we can approve this one. But first, let's investigate if this problem got introduced here.

**Review by dvaldivia — APPROVED** [review]
Tested. LGTM.

**Review by prakashsvmx — APPROVED** [review]
LGTM :+1:

**Pull request merged**